zoo/policies/cross-rl-agent/readme.md

```diff
@@ -0,0 +1,17 @@
+# cross-rl-agent for behavior model
```

Comment:
We will need to explain how to generate the following binaries in this readme as well since we cannot have un-sourced binaries in the repository:

```
./zoo/policies/cross-rl-agent/cross_rl_agent/models/Soc_Mt_TD3Network.ckpt.meta
./zoo/policies/cross-rl-agent/cross_rl_agent/models/Soc_Mt_TD3Network.ckpt.index
./zoo/policies/cross-rl-agent/cross_rl_agent/models/Soc_Mt_TD3Network.ckpt.data-00000-of-00001
./zoo/policies/cross-rl-agent/cross_rl_agent/train/trained model & scenarios/good model/Soc_Mt_TD3Network.ckpt.meta
./zoo/policies/cross-rl-agent/cross_rl_agent/train/trained model & scenarios/good model/Soc_Mt_TD3Network.ckpt.index
./zoo/policies/cross-rl-agent/cross_rl_agent/train/trained model & scenarios/good model/Soc_Mt_TD3Network.ckpt.data-00000-of-00001
```